ratis-common/src/main/java/org/apache/ratis/util/TimeoutScheduler.java

```diff
@@ -32,7 +31,7 @@
 import java.util.function.Consumer;
 import java.util.function.Supplier;
 
-public final class TimeoutScheduler implements Closeable {
+public final class TimeoutScheduler implements TimeoutExecutor {
```

Comment:
Should we remove this class, it will cause thread overflow and should not be called again

Reply:
Let's keep it for a while.  Not sure if we may find some problem in the new TimeoutTimer in the future.  It may be useful then.